EU4toV2/Source/Configuration.cpp

```diff
@@ -109,6 +117,11 @@ void Configuration::instantiate(std::istream& theStream, bool (*DoesFolderExist)
 	parseStream(theStream);
 	clearRegisteredKeywords();
 	setOutputName();
+	if (isHpmEnabled())
+	{
+		Vic2Path = Vic2ModPath + "/HPM";
+		Vic2DocumentsPath += "/HPM";
```

Comment:
Vic2DocumentsPath isn't used outside of verifying the folder exists and tests, so you don't need to update it. And even if you did, I don't think adding HPM to it is what you'd want.